app/src/main/java/org/sil/storyproducer/model/StoryIO.kt

```diff
@@ -82,9 +97,60 @@ fun storyFromJson(context: Context, storyTitle: String): Story?{
         // (i.e., to view message during debug, create a Logcat filter for "Log Message:"
         // looking for "SP::")
         Timber.e("SP::(%s)", errInfo) // uses Kotlin Log class with severity level: Error
+    }
+    // if we get here we caught an exception and have exited from the catch clause
+    // Even though we cannot parse the file with this version of the software, save the
+    // content of file for later evaluation for data salvage or error evaluation
+
+    // create a backup file name  with a time stamp
+    // example file name: storyWithParseErr_2021-06-03-14-11-26.json
+    var backupFileName =
+            "project/storyWithParseErr_" +  //location under directory root (eg "002 Lost Coin" )
+                    SimpleDateFormat("yyyy-MM-dd-HH-mm-ss").format(Date()) +
+                    ".json"
```

Comment:
We'll ask users to send this file to the developers. So I'm thinking that including the story name in the backupFileName adds some clarity for us when we receive it. Or are you seeing another way we get that info?